pkgs/development/ocaml-modules/mirage-random-test/default.nix

```diff
@@ -9,7 +12,7 @@ buildDunePackage rec {
   minimumOCamlVersion = "4.06";
 
   # due to cstruct
```

Comment:
Does this comment make sense anymore?

Reply:
IMO, as the PR is large, would be best to focus on fixing actual Dune 3 downstream packages issues, avoiding matters that are specific to packages and unrelated to main goal.

Reply:
In this particular case, there was a change that might of made the comment redundant, but GitHub doesn't seem to show that correctly any longer.